backend/census_historical_migration/workbooklib/end_to_end_core.py

```diff
@@ -260,8 +271,21 @@ def run_end_to_end(user, dbkey, year, result):
 
             result["success"].append(f"{sac.report_id} created")
     except Exception as exc:
-        tb = traceback.extract_tb(sys.exc_info()[2])
-        for frame in tb:
-            print(f"{frame.filename}:{frame.lineno} {frame.name}: {frame.line}")
+        error_type = type(exc)
```

Comment:
Exception chaining will be more readable. 
```
except KeyboardInterrupt:
    raise
except ValidationError as exc:
    logger.error(f"ValidationError: {exc}")
except DataMigrationError as exc:
    logger.error(f"DataMigrationError: {exc.message}")
except Exception as exc:
    logger.exception(f"Unexpected error type {type(exc).__name__}: {exc}")
```

Reply:
Done!